src/pages/MyRequests.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <q-page>
+    <header>
+      <h1>Meine Anzeigen</h1>
```

Comment:
Could you please use the i18n?
https://quasar.dev/options/app-internationalization
I have used this already